ledger/catchpointfilewriter_test.go

```diff
@@ -959,6 +958,169 @@ func TestCatchpointAfterTxns(t *testing.T) {
 	}
 }
 
+func TestCatchpointAfterStakeLookupTxns(t *testing.T) {
```

Comment:
Before making the catchpoint changes, was this scenario going to fail or worse be indeterministic?